app/Command/IDE.hs

```diff
@@ -19,7 +19,7 @@
 {-# LANGUAGE TemplateHaskell       #-}
 {-# LANGUAGE NoImplicitPrelude     #-}
 
-module Main where
+module Command.IDE (command) where
```

Comment:
Would prefer `Ide` like in the rest of the codebase, just for consistencies sake.